webrender/res/clip_shared.glsl

```diff
@@ -5,6 +5,8 @@
 
 flat varying vec4 vClipRect;
 flat varying vec4 vClipRadius;
+flat varying vec4 vClipMaskUvRect;
+flat varying vec4 vClipMaskScreenRect;
```

Comment:
We should probably rename this to be something like vClipMaskLocalRect, since (I think) it's in the local space of the primitive rectangle?